fe/fe-core/src/main/java/com/starrocks/sql/ast/ExpressionPartitionDesc.java

```diff
@@ -8,23 +8,41 @@
 
 public class ExpressionPartitionDesc extends PartitionDesc {
 
+    //must be column
     private SlotRef slotRef;
-    //expr in outputExpression
+    //must be type of SlotRef or FunctionCallRef
     private Expr expr;
+    // if expression is type of FunctionCallExpr, isFunc = true
+    // else isFunc = false
+    private boolean isFunction;
```

Comment:
is it necessary?

Reply:
just help check expr type easily,  I used in MaterializedViewAnalyzer, can replace with `expr instanceof FunctionCallExpr`